src/java/org/apache/cassandra/service/accord/AccordJournal.java

```diff
@@ -403,7 +418,7 @@ public void purge(CommandStores commandStores)
     public void replay(CommandStores commandStores)
     {
         journal.closeCurrentSegmentForTestingIfNonEmpty();
-        try (AccordJournalTable.KeyOrderIterator<JournalKey> iter = journalTable.readAll())
+        try (AccordJournalTable.KeyOrderIterator<JournalKey> iter = journalTable.readAll(AccordJournalTable.ReadAllIntent.Reply))
```

Comment:
this is less critical after the rebase as the `checkAll` function got removed... when we are reading for validation we don't want to populate the index... I left this here to be explicit about this assumption but I can remove.